src/mini_components/react_component.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
Consider using camelCase for files and folders as it's a more conventional in React.js community and in JS in general.